tests/Symfony/Installer/Tests/NewCommandTest.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+/*
+ * This file is part of the Symfony Installer package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Installer\Tests;
+
+use Distill\Chooser;
+use Distill\Distill;
+use Distill\Exception\IO\Input\FileCorruptedException;
+use Distill\Exception\IO\Input\FileEmptyException;
+use Distill\Exception\IO\Output\TargetDirectoryNotWritableException;
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Event\EmitterInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Installer\NewCommand;
+
+/**
+ * @runTestsInSeparateProcesses
+ */
+class NewCommandTest extends CommandTest
+{
+    /**
+     * @var Distill|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $distill;
+
+    /**
+     * @var Chooser|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $chooser;
+
+    /**
+     * @var ClientInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $client;
+
+    /**
+     * @var EmitterInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $emitter;
+
+    private $projectsDir;
+
+    protected function setUp()
+    {
+        $this->chooser = $this->getMockBuilder('Distill\Chooser')
```

Comment:
mocking classes from other libraries don't ensure that our code works, because the other libraries might change its implementation and mocks will then be out of sync. You should not mock accross boundaries.